build.gradle.kts

```diff
@@ -188,15 +199,15 @@ subprojects {
 
         dependencies {
             "commonMainImplementation"(kotlin("stdlib-common"))
-            "commonMainImplementation"("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.2-native-mt")
+            "commonMainImplementation"("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3-native-mt")
```

Comment:
is it a good idea to move the version to `gradle.properties` so it'd be possible to change it in one place?